app/assess/data.py

```diff
@@ -342,3 +346,45 @@ def get_comments(application_id: str, sub_criteria_id: str):
     )
     comment_response = get_data(comment_endpoint)
     return comment_response
+
+def get_file_url(filename: str, application_id: str):
+    """_summary_: Function is set up to retrieve
+    files from aws bucket.
+    Args:
+        filename: Takes an filename
+        application_id: Takes an application_id # noqa
+    Returns:
+        Returns a presigned url.
+    """
+
+    if (filename == None):
+        return None
+
+    if "VCAP_SERVICES" in os.environ:
+    # Extract the JSON string from the environment variable
+        vcap_services = json.loads(os.environ["VCAP_SERVICES"])
+
+        # Check for the "aws-s3-bucket" service
+        if "aws-s3-bucket" in vcap_services:
+            # Extract the credentials from the service's configuration
+            s3_credentials = vcap_services["aws-s3-bucket"][0]["credentials"]
+            Config.AWS_ACCESS_KEY_ID = s3_credentials["aws_access_key_id"]
+            Config.AWS_SECRET_ACCESS_KEY = s3_credentials["aws_secret_access_key"]
+            Config.AWS_BUCKET_NAME = s3_credentials["bucket_name"]
```

Comment:
It might be worth performing this mutation on the module itself rather than each time this function is executed, if it's scoped to the module I think it'll be executed on the first import only